README.md

```diff
@@ -162,13 +189,17 @@ dev_dependencies     | map<text, text> | DevDeps of package
 peer_dependencies    | map<text, text> | peerDeps of package
 optional_dependencies| map<text, text> | any optional dependencies
 
+## Test
+
+Ensure you have [Cassandra] running local.
 
-## Tests
 ```sh
 npm test
 ```
 
-## License
-MIT
-
+[wrhs]: https://registry.npmjs.org/warehouse.ai/latest
```

Comment:
1. there's already a link to the github of this, do we need a link to npm also?
1. this link is called `wrhs` should it be linking to the CLI of the same name?

Reply:
The idea here is to demonstrate an example `package.json` for developers unaware of how those would look and what other properties there are generally on that. The links have distinct different goal and I'd rather demo the `package.json` of the main repo than the CLI. But that said not feeling strong about it.